tests/basic.test/runit

```diff
@@ -330,4 +338,42 @@ cdb2sql ${CDB2_OPTIONS} $dbnm default 'exec procedure sys.cmd.send("sql hist")'
 res=`cdb2sql --tabs ${CDB2_OPTIONS} $dbnm default "select 1"`
 assertres $res 1
 
+echo "create and use UNIQUE INDEX with multiple NULL values"
```

Comment:
I thought about doing that; however, given the overhead involved in creating an entirely new test file and the speed at which these new tests will execute, it seemed like overkill.

Reply:
But it would give us a better idea of what the test is testing for.

Reply:
Might I suggest we add to a different existing test, such as sql.test/, which would allow us to test this feature as a separate test (and document in the README for that test), and also keep basic simple and short.

Reply:
Good idea, doing that now.